scripts/deploy/images/Dockerfile

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I am assuming this and `entrypoint.sh` are used to create the `azstor-prereq-linux` image? If so, it would be good to put these in a subdirectory. That way, we can add additional image builds as necessary without confusion.

Reply:
Let's see if we can make the other approach you mentioned work. Otherwise, I'll move it to a different folder.

Reply:
Worked like a charm. Deleted these files.